zebra-chain/src/work/difficulty/tests/vectors.rs

```diff
@@ -457,7 +457,7 @@ fn check_testnet_minimum_difficulty_block(height: block::Height) -> Result<(), R
             .signed_duration_since(previous_block.header.time);
 
         // zcashd requires a gap that's strictly greater than 6 times the target
-        // threshold, but ZIP-205 and ZIP-208 are ambiguous. See bug #1276.
+        // threshold.
```

Comment:
These ZIPs got fixed in response to our questions:
```suggestion
        // threshold, as documented in ZIP-205 and ZIP-208:
        // https://zips.z.cash/zip-0205#change-to-difficulty-adjustment-on-testnet
        // https://zips.z.cash/zip-0208#minimum-difficulty-blocks-on-testnet
```